SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -418,3 +410,43 @@ void USpatialWorkerConnection::CacheWorkerAttributes()
 		CachedWorkerAttributes.Add(UTF8_TO_TCHAR(Attributes->attributes[Index]));
 	}
 }
+
+USpatialNetDriver* USpatialWorkerConnection::GetSpatialNetDriverChecked() const
```

Comment:
`USpatialNetDriver* GetSpatialNetDriverChecked() const;` is missing from the header file